src/content/language/extension-types.md

```diff
@@ -446,8 +446,8 @@ and when testing against an extension type (`case MyExtensionType(): ... `).
 void main() {
   var n = NumberE(1);
 
-  // Run-time type of 'n' is representation type 'int'.
-  if (n is int) print(n.value); // Prints 1.
+  // The run-time type of 'n' is the representation type 'int'.
+  if (n is int) print(n); // Prints 1.
```

Comment:
I changed `n.value` to `n` because it has the same behavior, and `n.value` raises a question which is slightly tricky to explain, and it's not the topic of this text.

The tricky part is that `n` is not promoted to `int` even though we have just tested that `n is int` and the result was `true`. This means that `n` still has the static type `NumberE` when we reach the expression `n.value`. This might very well be confusing to a reader who does not consider the fact that `int` is not a subtype of `NumberE`. A reader who thinks that the "promotion" has occurred will not understand why `n.value` isn't a compile-time error. With `print(n)`, the code works whether or not `n` has been promoted, so we just avoid talking about promotion entirely.

Reply:
Sounds good to me. @parlough any thoughts on this?